autoInstaller.sh

```diff
@@ -297,12 +301,14 @@ if $installRPMS; then
 fi
 
 if $installgtm || $installYottaDB ; then
+    gtmInst=true
```

Comment:
this seems odd, just use the existing environment variables $installgtm || $installYottaDB